explainer.md

```diff
@@ -542,6 +542,16 @@ There is a related effort to expose motion-sensing controllers through the Gamep
 ### These alternatives don’t account for presentation
 It’s important to realize that all of the alternative solutions offer no method of displaying imagery on the headset itself, with the exception of Cardboard-like devices where you can simply render a fullscreen split view. Even so, that doesn’t take into account how to communicate the projection or distortion necessary for an accurate image. Without a reliable presentation method the ability to query inputs from a headset becomes far less valuable.
 
+### What's the deal with WebVR?
+
+There's understandably some confusion between the WebXR and an API that some browsers have implemented at various points in the past called WebVR. Both handle communication with Virtual Reality hardware, and both have very similar names. So what's the difference between these two APIs?
+
+**WebVR** was an API developed in the earliest days of the current generation of Virtual Reality hardware/software, started around the time that the Oculus DK2 was announced. Native VR APIs were still in their formative stages, and the capabilities of commercial devices were still being determined, and as such the WebVR API developed around some assumptions that would not hold true long term. For example, the API assumed that applications would always need to render a single left and right eye view of the scene, that the separation between eyes would only ever involve translation and not rotation, and that only one cannonical tracking space was necessary to support. In addition, the API design made forward compatibility with newer device types, like mobile AR, difficult, to the point that it may have necessitated a separate API. WebVR also made some questionable descisions regarding integration with the rest of the web platform, specifically in terms of how it interacted with WebGL and the Gamepad API. Despite this, it worked well enough in the short term that some UAs, especially those shipped specifically for VR devices, decided to ship the API to their users.
```

Comment:
Nits...

>**WebVR** was an API developed in the earliest days of the current generation of Virtual Reality hardware/software, started around the time that the Oculus DK2 was announced

"started" should maybe be "starting"?

>Native VR APIs were still in their formative stages, and the capabilities of commercial devices were still being determined, and as such the WebVR API developed around some assumptions that would not hold true long term.

Slightly strange "and" chaining.  Perhaps split the "and as such..." into its own sentence?